src/components/formik/FormikComponent.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import * as Yup from "yup";
 import { AnyObject } from "yup/lib/types";
-
+import { Link,useHistory,Router} from "react-router-dom";
```

Comment:
Please remove `Link` and `Router` imports since they are not used.
```suggestion
import { useHistory } from "react-router-dom";
```